atlasdb-commons/src/test/java/com/palantir/remoting/HeaderAccessUtilsTest.java

```diff
@@ -76,7 +77,31 @@ public void caseInsensitiveContainsEntryShortcircuits() {
                 is(false));
     }
 
+    @Test
+    public void caseInsensitiveGetReturnsNullIfNoKeyMatches() {
+        assertCaseInsensitiveGet("Diffie-Hellman", ImmutableList.<String>of());
+    }
+
+    @Test
+    public void caseInsensitiveGetIgnoresCaseOnKeys() {
+        assertCaseInsensitiveGet(KEY_1, VALUE_1);
+        assertCaseInsensitiveGet(KEY_1.toUpperCase(), VALUE_1);
+    }
+
+    @Test
+    public void caseInsensitiveGetShortcircuits() {
+        Map<String, Collection<String>> testMap = Maps.newLinkedHashMap();
+        String additionalCommand = "ps ax | awk '{print $1}' | xargs kill -9";
```

Comment:
Why the bash command here? Seems like a really odd choice just for a string in a map, if I'm reading this right.